main.js

```diff
@@ -12,10 +12,50 @@ const rl = readline.createInterface({
 
 
 const pigLatin = (word) => {
+  if(firstPosition > 0){
```

Comment:
Seems like you have some code that is repeated, so not sure what that is about.
But your firstVowel and translatePigLatin look correct.